Domain/src/main/java/starterProject/App.java

```diff
@@ -38,5 +39,7 @@ public static void main(String[] args) {
 
     }
 
+   public static List<Car> lists = createCar();
+
```

Comment:
This might seem pedantic but it's a good idea to make sure we don't have excess whitespace and consistent code style for readability.